stdlib/3/unittest/util.pyi

```diff
@@ -0,0 +1,29 @@
+from typing import Any, List, Sequence, Tuple, TypeVar
+
+_T = TypeVar('_T')
+_Mismatch = Tuple[_T, _T, int]
+
+_MAX_LENGTH: int = ...
+_PLACEHOLDER_LEN: int = ...
+_MIN_BEGIN_LEN: int = ...
+_MIN_END_LEN: int = ...
+_MIN_COMMON_LEN: int = ...
+_MIN_DIFF_LEN: int = ...
+
+def _shorten(s: str, prefixlen: int, suffixlen: int) -> str: ...
+
+def _common_shorten_repr(*args: Any) -> Tuple[str]: ...
+
+def safe_repr(obj: Any, short: bool = ...) -> str: ...
```

Comment:
`Any` -> `object`, since this function literally takes any object. `Any` should be used for types that can't be properly represented in the current type system.